examples/kubernetes/manifests/registry-storage.yml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: registry-storage
+  labels:
+    storage: registry
+spec:
+  capacity:
+    storage: 20Gi
+  accessModes:
+    - ReadWriteMany
```

Comment:
Yeah, this is something to improve (I just blindly gave access to everything to make sure that it works 😁)